<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8"/>
    <meta property="og:url" content="https://www.eurekalert.org/pub_releases/2019-11/buso-ncc111519.php"/>
    <meta property="og:site_name" content="EurekAlert!"/>
    <meta property="article:published_time" content="2019-11-15T05:00:00+00:00"/>
    <meta property="og:title" content="New candidate cancer genes identified using math models"/>
    <meta property="og:description" content="Computational modeling is the use of computers to simulate and study the behavior of complex systems. Computational approaches are widely adopted in the bioimedical sciences and can be used to sift through large volumes of complex data to extract recurrent patterns that may point to a disease's causes and effects."/>
  </head>
  <body>
    <article>
      <h1>New candidate cancer genes identified using math models</h1>
      <h2>Results available to the research community via web-based portal</h2>
      <address><time datetime="2019-11-15T05:00:00+00:00">15 Nov 2019, 05:00</time> by <a rel="author">Boston University School of Medicine</a></address>
      <p>(Boston)--Computational modeling is the use of computers to simulate and study the behavior of complex systems. Computational approaches are widely adopted in the bioimedical sciences and can be used to sift through large volumes of complex data to extract recurrent patterns that may point to a disease's causes and effects.</p>
      <p>Researchers from Boston University School of Medicine (BUSM) have developed a novel computational method, integration of Epi-DNA and Gene Expression (iEDGE), whose application to the analysis of more than 8,500 tumor profiles from The Cancer Genome Atlas has led to the discovery of genes whose alteration (mutation or copy number alteration) may contribute to cancer susceptibility. This breakthrough may lead to new therapeutic targets for numerous cancers.</p>
      <p>According to the researchers, iEDGE identified several candidate breast cancer drivers, including RBM17 (a splicing factor amplified in Triple-Negative Breast Cancer) and SIRT3 (a candidate tumor suppressor and a promising therapeutic target). It also identified multiple candidate pan-cancer drivers, including TRIP13 (previously shown to promote tumor growth in colorectal cancer and a predictor of poor prognosis in prostate cancer), ORAOV1 (a gene overexpressed in many solid tumors), and TPX2 (a potent oncogene amplified in many cancers and a promising therapeutic target), among others.</p>
      <p>"While further functional studies will be needed to evaluate the therapeutic relevance of our findings, these results study show the efficacy of iEDGE at identifying candidate drivers and potentially novel targets for therapy," explained corresponding author Stefano Monti, PhD, associate professor of medicine at BUSM.</p>
      <p>The open source tool iEDGE is freely downloadable at github.com/montilab/iEDGE and biomedical scientists are able to apply it to the analysis of their own data to advance their research. As a companion to the published findings, a web-based portal for the interactive querying and visualization of the study's results is hosted at montilab.bu.edu/iEDGE</p>
      <p>"Through the web-based portal, all the data and results of our pan-cancer analysis are accessible to the research community, who can search for gene candidates and for their potential mechanisms of action, and thus support their translational research toward more effective cancer treatments," added first author Amy Li, PhD, a graduate from the Boston University Bioinformatics PhD program.</p>
      <hr/>
      <p>These findings appear online in the journal <i>Scientific Reports</i>.</p>
      <p>Funding for this study was provided in part by <a href="http://findthecausebcf.org/">Find the Cause Breast Cancer Foundation</a>, Boston University Superfund Research Program (P42 ES007381-21), and the National Institute on Aging (NIA cooperative agreements U19-AG023122, and R01AG061844).</p>
    </article>
  </body>
</html>