<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8"/>
    <meta property="og:url" content="https://www.nature.com/articles/d41586-019-03392-9"/>
    <meta property="og:site_name" content="Nature"/>
    <meta property="article:published_time" content="2019-11-11T00:00:00+00:00"/>
    <meta property="og:title" content="CRISPR tool modifies genes precisely by copying RNA into the genome"/>
    <meta property="og:description" content="A ‘search-and-replace’ method for editing the genome."/>
  </head>
  <body>
    <article>
      <h1>CRISPR tool modifies genes precisely by copying RNA into the genome</h1>
      <h2>The ultimate goal of genome editing is to be able to make any specific change to the blueprint of life. A ‘search-and-replace’ method for genome editing takes us a giant leap closer to this ambitious goal.</h2>
      <address><time datetime="2019-11-11T00:00:00+00:00">11 Nov 2019</time> by <a rel="author">Randall J. Platt</a></address>
      <p>Variation in the DNA sequences that constitute the blueprint of life is essential to the fitness of any species, yet thousands of DNA alterations are thought to cause disease. After decades of research in genetics and molecular biology, tremendous progress has been made in developing genome-editing tools for correcting such alterations. But a seemingly fundamental limit to the efficiency and precision of gene editing was reached, owing to the tools’ reliance on complex and competing cellular processes. <a href="https://www.nature.com/articles/s41586-019-1711-4">Writing in <i>Nature</i></a>, Anzalone <i>et al.</i> [<sup><a href="https://www.nature.com/articles/d41586-019-03392-9#ref-CR1">1</a></sup>] describe ‘search-and-replace’ genome editing, in which the marriage of two molecular machines enables the genome to be altered precisely. The technique has immediate and profound implications for the biomedical sciences.</p>
      <p>Human efforts to engineer genomes pre-date knowledge of genes or even of the source of heredity. The first genome engineering relied on natural variation and artificial selection through selective breeding. Modern maize (corn), for example, was ‘engineered’ from its wild ancestor, teosinte, through artificial selection more than 9,000 years ago [<sup><a href="https://www.nature.com/articles/d41586-019-03392-9#ref-CR2">2</a></sup>]. Later progress was fuelled by the realization that DNA sequences shape life, and that evolution can be augmented and artificially accelerated through the use of mutagenic agents, such as radiation or chemicals.</p>
      <p>Next came the discovery that cellular processes for repairing mistakes in DNA sequences could be hijacked, allowing sequences from a foreign ‘template’ DNA to be inserted into the genome at DNA breaks [<sup><a href="https://www.nature.com/articles/d41586-019-03392-9#ref-CR3">3</a></sup>]. This process is greatly enhanced if the DNA is intentionally damaged [<sup><a href="https://www.nature.com/articles/d41586-019-03392-9#ref-CR4">4</a></sup>] [<sup>,</sup>] [<sup><a href="https://www.nature.com/articles/d41586-019-03392-9#ref-CR5">5</a></sup>] — a finding that sparked a search of more than 20 years for an enzyme that could specifically cut DNA at locations of interest. The search culminated in the adoption of the bacterial CRISPR–Cas9 system, in which the enzyme Cas9 uses a customizable RNA guide to search for DNA sequences to cut in human cells [<sup><a href="https://www.nature.com/articles/d41586-019-03392-9#ref-CR6">6</a></sup>] [<sup>–</sup>] [<sup><a href="https://www.nature.com/articles/d41586-019-03392-9#ref-CR8">8</a></sup>] (Fig. 1a).</p>
      <figure>
        <img src="https://media.nature.com/w800/magazine-assets/d41586-019-03392-9/d41586-019-03392-9_17372794.png"/>
        <figcaption><b>Figure 1 | Evolution of genome editing.</b> <b>a</b>, In conventional genome editing, a Cas9 enzyme is directed to a position in the genome by a guide RNA, and produces a double-strand break. The host cell’s DNA-repair machinery fixes the break, guided by a template DNA, incorporating template sequences into the duplex. <b>b</b>, In an approach called base editing, a Cas9 that produces only single-strand breaks (nicks) works with a deaminase enzyme. The deaminase chemically modifies a specific DNA base — here, a cytidine base (C) is converted to uracil (U). DNA repair then fixes the nick and converts a guanine–uracil (G–U) intermediate to an adenine–thymine (A–T) base pair. This method is more precise than <b>a</b>, but makes only single-nucleotide edits. <b>c</b>, Anzalone <i>et al</i>. [<sup><a href="https://www.nature.com/articles/d41586-019-03392-9#ref-CR1">1</a></sup>] report prime editing, which can precisely edit DNA sequences. A nick-producing Cas9 and a reverse transcriptase enzyme produce nicked DNA into which sequences corresponding to the guide RNA have been incorporated. The original DNA sequence is cut off, and DNA repair then fixes the nicked strand to produce a fully edited duplex. In some cases, another nick is made in the unedited strand of the duplex before the DNA-repair step (not shown).</figcaption>
      </figure>
      <p>CRISPR–Cas9 sparked a revolution in the biomedical sciences by making genome editing accessible to all researchers, but, ultimately, it is just a fancy pair of molecular scissors that cuts DNA. Because cuts in DNA are deadly to cells, they are urgently repaired by one of many independent pathways. In the context of genome editing, the desired outcome is for repair to be directed by a template DNA, resulting in precise edits. But most cells prefer to use an alternative mechanism, in which the DNA template is ignored and the two broken ends of DNA are imperfectly stitched back together — a major limitation for genome editing.</p>
      <p>Much effort over the past few years has focused on shifting the balance from imperfect to precise editing. One effective strategy is to edit DNA without cutting both DNA strands in the helix — double-strand breaks are the main insult that leads to imperfect edits. A milestone in this regard was the development of base editing [<sup><a href="https://www.nature.com/articles/d41586-019-03392-9#ref-CR9">9</a></sup>], a process in which a version of the Cas9 enzyme that cuts only one DNA strand is combined with an enzyme that can switch one specific DNA base for another, near the nick site (Fig. 1b). However, the technical constraints of base editing, and the need to modify more than just single DNA bases, meant that new genome-editing approaches were still desperately needed.</p>
      <p>Anzalone and co-workers now largely fill this need with a technique called prime editing. Their approach relies on a hybrid molecular machine consisting of a modified version of Cas9, which cuts only one of the two DNA strands, and a reverse transcriptase enzyme, which installs new and customizable DNA at the cut site (Fig. 1c). This marriage parallels a naturally occurring process in yeast, in which DNA that corresponds to an RNA sequence is incorporated into the genome by a reverse transcriptase [<sup><a href="https://www.nature.com/articles/d41586-019-03392-9#ref-CR10">10</a></sup>].</p>
      <p>The prime-editing process is orchestrated by an engineered, two-part RNA guide. The ‘search’ part of the guide directs Cas9 to a specific sequence in the DNA target, where it cuts one of the two DNA strands. The reverse transcriptase then produces DNA complementary to the sequence in the ‘replace’ part of the RNA guide, and installs it at one of the cut DNA ends, where it takes the place of the original DNA sequence.</p>
      <p>At this point, the duplex DNA being modified consists of two non-complementary strands: the edited strand, and the intact strand that wasn’t cut by Cas9. Non-complementary sequences are not tolerated in cells, so one of the strands must be fixed by DNA-repair processes to match the other, with the intact strand typically being preferentially retained. The authors therefore usually had to use a second RNA guide to direct a cut to the intact strand, to increase the chances that that strand would be repaired to match the edited sequence. The cut must be made strategically to avoid breaking both strands at the same time or place.</p>
      <p>Anzalone <i>et al</i>. demonstrate the versatility of prime editing by using it to efficiently and precisely install a wide range of sequences into DNA. For example, they used it <i>in vitro</i> in human embryonic kidney cells to correct the mutations that give rise to the blood disorder sickle-cell disease, and to edit the mutations that cause the neurological condition Tay–Sachs disease. Imperfect edits were almost entirely avoided. The authors also carried out edits in human cancer cells and in mouse neurons<i> in vitro</i>.</p>
      <p>For decades, the potential of genome editing has been constrained by the difficulty of making precise modifications, and so applications have focused heavily on situations in which imperfect DNA edits are useful. For example, such edits can be used to impair the function of a gene, providing an avenue for understanding its function [<sup><a href="https://www.nature.com/articles/d41586-019-03392-9#ref-CR11">11</a></sup>]. Prime editing now makes it faster and easier than before to install or correct one or many specific mutations (such as those found in human patients, or synthetic sequences that are useful for research purposes). And it makes more cell types available for manipulation than was previously possible. The chains that have shackled gene editing have thus come off — no doubt quickening the pace of research and enabling a list of new applications.</p>
      <p>Nevertheless, prime editing has limitations. First, the sophisticated, multi-step molecular dance that occurs between the prime-editing components is not yet predictable and doesn’t always turn out as intended. Imperfect random edits can therefore still arise, which means that several combinations of components might need to be tested, to work out the choreographies required for each edit of interest. Second, delivering the large prime-editing system into some cell types could be challenging, given that many previous attempts have faltered with the conventional Cas9 system [<sup><a href="https://www.nature.com/articles/d41586-019-03392-9#ref-CR12">12</a></sup>], which is roughly half the size.</p>
      <p>For research purposes, these limitations are mostly just inconvenient, and will probably be overcome through follow-up work directed at better understanding and fine-tuning the method. For medical applications, however, these issues present a much greater challenge — imperfect DNA edits are unacceptable, and efficient delivery of the prime-editing system to cells will be crucial. So although prime editing certainly has the potential to give us unprecedented control over the blueprint of life, only time will tell whether it becomes just another tool in the CRISPR toolbox or a cure-all for genetic diseases.</p>
      <p>doi: 10.1038/d41586-019-03392-9</p>
      <h3>References</h3>
      <p>1. <anchor name="ref-CR1"/>Anzalone, A. V. <i>et al.</i> <i>Nature</i> https://doi.org/10.1038/s41586-019-1711-4 (2019).</p>
      <p>2. <anchor name="ref-CR2"/>Matsuoka, Y. <i>et al.</i> <i>Proc. Natl Acad. Sci. USA</i> <b>99</b>, 6080–6084 (2002).</p>
      <p>3. <anchor name="ref-CR3"/>Capecchi, M. R. <i>Science</i> <b>244</b>, 1288–1292 (1989).</p>
      <p>4. <anchor name="ref-CR4"/>Rudin, N., Sugarman, E. &amp; Haber, J. E. <i>Genetics</i> <b>122</b>, 519–534 (1989).</p>
      <p>5. <anchor name="ref-CR5"/>Rouet, P., Smih, F. &amp; Jasin, M. <i>Mol. Cell. Biol.</i> <b>14</b>, 8096–8106 (1994).</p>
      <p>6. <anchor name="ref-CR6"/>Jinek, M. <i>et al.</i> <i>Science</i> <b>337</b>, 816–821 (2012).</p>
      <p>7. <anchor name="ref-CR7"/>Cong, L. <i>et al.</i> <i>Science</i> <b>339</b>, 819–823 (2013).</p>
      <p>8. <anchor name="ref-CR8"/>Mali, P. <i>et al.</i> <i>Science</i> <b>339</b>, 823–826 (2013).</p>
      <p>9. <anchor name="ref-CR9"/>Komor, A. C., Kim, Y. B., Packer, M. S., Zuris, J. A. &amp; Liu, D. R. <i>Nature</i> <b>533</b>, 420–424 (2016).</p>
      <p>10. <anchor name="ref-CR10"/>Keskin, H. <i>et al.</i> <i>Nature</i> <b>515</b>, 436–439 (2014).</p>
      <p>11. <anchor name="ref-CR11"/>Platt, R. J. <i>et al.</i> <i>Cell Rep.</i> <b>19</b>, 335–350 (2017).</p>
      <p>12. <anchor name="ref-CR12"/>Cox, D. B. T., Platt, R. J. &amp; Zhang, F. <i>Nature Med.</i> <b>21</b>, 121–131 (2015).</p>
      <related>
        <a href="http://www.nature.com/articles/s41586-019-1746-6"/>
        <a href="http://www.nature.com/articles/s41586-019-1753-7"/>
        <a href="http://www.nature.com/articles/s41586-019-1749-3"/>
        <a href="http://www.nature.com/articles/s41586-019-1728-8"/>
        <a href="http://www.nature.com/articles/d41586-019-03490-8"/>
        <a href="http://www.nature.com/articles/d41586-019-03486-4"/>
        <a href="http://www.nature.com/articles/d41586-019-03416-4"/>
        <a href="http://www.nature.com/articles/s41586-019-1732-z"/>
      </related>
    </article>
  </body>
</html>