<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8"/>
    <meta property="og:url" content="https://www.eurekalert.org/pub_releases/2019-11/asu-amo112519.php"/>
    <meta property="og:site_name" content="EurekAlert!"/>
    <meta property="article:published_time" content="2019-11-25T05:00:00+00:00"/>
    <meta property="og:title" content="Aquatic microorganisms offer important window on the history of life"/>
    <meta property="og:description" content="Jeremy Wideman and his colleagues, including Prof. Thomas Richards at the University of Exeter describe a new method for investigating the genomes of eukaryotic flagellate organisms. These elusive aquatic creatures may offer clues about LECA--the progenitor of all complex life on earth."/>
  </head>
  <body>
    <article>
      <h1>Aquatic microorganisms offer important window on the history of life</h1>
      <address><time datetime="2019-11-25T05:00:00+00:00">25 Nov 2019, 05:00</time> by <a rel="author">Arizona State University</a></address>
      <p>The air, earth and water of our planet are pulsating with living things. Yet, a vast and diverse web of life exists, about which almost nothing is known. This is the world of flagellates, tiny organisms that persist in staggering numbers in many diverse ecosystems around the world.</p>
      <p>According to Jeremy Wideman, a researcher at the Biodesign Center for Mechanisms in Evolution at Arizona State University, we have a great deal to learn from these delicate and wildly varied creatures. Among other surprises, flagellates could provide valuable clues about a shadowy event that may have occurred 1.5-2 billion years ago, (no one is really sure of the timing), with the arrival of a new type of cell.</p>
      <p>Known as LECA, it was a sort of primal egg out of which the astonishing profusion of complex life--from flagellate organisms, fungi and plants, to insects, zebra, and humans, exploded and spread over the earth.</p>
      <p>In new research appearing today in the journal Nature Microbiology, Wideman and his colleagues, including Prof. Thomas Richards at the University of Exeter describe a new method for investigating the genomes of eukaryotic flagellate organisms, which have been notoriously tricky to pinpoint and sequence.</p>
      <p>Specifically, they explored samples of mitochondrial DNA, sequencing around 100 such genomes for previously undocumented flagellates. The new technique could help scientists like Wideman begin to fill in the largely blank region of the eukaryotic puzzle, where flagellate life flourishes.</p>
      <p>
        <b>Cellular worlds</b>
      </p>
      <p>Wideman, originally a traditional cell biologist, became frustrated with the many unaddressed questions in the field, recently joining the emerging discipline of evolutionary cell biology. This rapidly advancing research area uses cells as fundamental units for the study of evolutionary processes and imports concepts from evolutionary biology to better understand how cells work. "I'm literally a cell biologist that wants to know more about things we know nothing about," he says.</p>
      <p>Evolutionary cell biology is a profoundly transdisciplinary endeavor, fusing evolutionary theory, genomics and cell biology with quantitative branches of biochemistry, biophysics, and population genetics.</p>
      <p>Flagellates include many parasites implicated in human disease, from the intestinal bug Giardia to more damaging trypanosomes, and leishmania. Flagellates also perform more benevolent tasks. As the major consumers of bacteria and other protists in aquatic ecosystems, they help ensure the recycling of limiting nutrients.</p>
      <p>Single-celled eukaryotic organisms, which include flagellates, constitute the overwhelming majority of eukaryotic diversity, vastly outpacing the more familiar multicellular plants, animals, and fungi. Despite their importance and ubiquity across the globe, flagellates are, as Wideman stresses, an almost entirely unknown inhabitant of the living world and one of the most enigmatic. When viewed under a microscope, their often science fiction-like appearance is markedly distinct from the kinds of eukaryotic cells commonly described in biology textbooks. Their emergence from comparatively rudimentary prokaryotes marks the most momentous transition in the history of life on earth.</p>
      <p>"Novel lineages of heterotrophic flagellates are being discovered at an alarming, rate," Wideman says. "In the last two years 2 kingdom level lineages have been discovered (see here and here), meaning lineages that have been evolving independently of animals and fungi for over a billion years." Nevertheless, researchers have barely scratched the surface of this astonishing diversity and new methods must be brought to bear to speed up the quest. (Heterotrophs are organisms that cannot synthesize their own food, relying instead on other organisms for nutrition.)</p>
      <p>
        <b>Microbial safari</b>
      </p>
      <p>Any drop of pond, lake or ocean water is likely to contain many flagellates, but separating them from a multitude of non-flagellates and accurately reading their genomes by conventional means has been slow and painstaking work. Only a minute fraction of extant flagellates have known genomic sequences and it's even possible that the overwhelming majority have never actually been seen. According to Wideman, flagellate life forms represent the 'dark matter' of the eukaryotic universe.</p>
      <p>"Heterotrophic flagellates are the target," Wideman says. "They're not a lineage. They're many, many lineages that are from all over the tree of life. LECA, the Last Eukaryotic Common Ancestor, was a heterotrophic flagellate, which means, that every major lineage (of eukaryotes) evolved from some sort of heterotrophic flagellate."</p>
      <p>To access the elusive flagellate mitochondrial DNA, the researchers exploited a feature common to all flagellates and from which they take their name--the existence of flagella, which, unlike in animal sperm are on the front of cells and are often used to pull them forward like a microscopic breast stroke but are also involved in sensation, feeding, and perhaps other, as-yet unknown functions.</p>
      <p>Flagella are rich in a particular protein known as tubulin. The new method for identifying flagellates and distinguishing them from their aquatic neighbors--primarily algae and bacteria--capitalizes on this fact by applying a selective stain to flagella-bearing organisms, activated by their high tubulin content. (Algal cells are naturally marked by their chloroplasts, which the flagellates of interest in the new study lack.)</p>
      <p>Samples of sea water collected in 2014 off the coast of California provided a test case. Using the technique, the researchers gathered a windfall of mitochondrial sequence data, significantly expanding the catalog of flagellates identified by molecular means. Indeed, they doubled the existing mitochondrial DNA library for flagellate organisms. "We got many, many different kinds of organisms. So it was a very rich sample and very few were identical," Wideman says.</p>
      <p>
        <b>In search of LECA</b>
      </p>
      <p>Apart from the mystery of life's origin, the puzzle of where eukaryotes came from and how the LECA event transpired is the most important and vexing unanswered question in all of biology. (It has been dubbed the black hole at the heart of the living world.)</p>
      <p>Correctly establishing the sequence of events underlying the crucial innovations within eukaryotes, from whence all complex life sprang, will take much more research in unexplored regions of the existing eukaryotic domain, particularly, the flagellates. Wideman believes the rapid advance of techniques for identifying and sequencing these organisms, such as the one outlined in the new study, offer hope such questions may one day find answers.</p>
      <hr/>
      <p>This project was supported by a Gordon and Betty Moore foundation grant (GBMF3307) to 597 TAR, AES, AZW and PJK, and a Philip Leverhulme Award (PLP-2014-147) to TAR. Field sampling was 598 supported by the David and Lucile Packard Foundation and GBMF3788 to AZW. TAR and AM are 599 supported by Royal Society University Research Fellowships. JGW was supported by the European 600 Molecular Biology Organization Long-term Fellowship (ALTF 761-2014) co-funded by European 601 Commission (EMBOCOFUND2012, GA-2012-600394) support from Marie Curie Actions and a College for 602 Life Sciences Fellowship at the Wissenschaftskolleg zu Berlin. RRM is supported by CONICYT FONDECYT 603 11170748. FM is supported by Genome Canada.</p>
      <p>Written by: Richard Harth <br/>Senior Science Writer: Biodesign Institute at ASU <br/><a href="mailto:richard.harth@asu.edu">richard.harth@asu.edu</a></p>
      <p>Written by: Richard Harth <br/>Senior Science Writer: Biodesign Institute at ASU <br/><a href="mailto:richard.harth@asu.edu">richard.harth@asu.edu</a></p>
    </article>
  </body>
</html>