<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8"/>
    <meta property="og:url" content="https://www.technologynetworks.com/genomics/news/a-highly-precise-cas9-enzyme-sacas9-hf-is-added-to-the-crispr-toolbox-326965"/>
    <meta property="og:site_name" content="Technology Networks"/>
    <meta property="article:published_time" content="2019-11-07T00:00:00+00:00"/>
    <meta property="og:title" content="A Highly Precise Cas9 Enzyme, SaCas9-HF, Is Added to the CRISPR Toolbox"/>
    <meta property="og:description" content="A team of scientists from City University of Hong Kong (CityU) and the Karolinska Institute has created a novel protein that can increase the target accuracy in genome editing."/>
  </head>
  <body>
    <article>
      <h1>A Highly Precise Cas9 Enzyme, SaCas9-HF, Is Added to the CRISPR Toolbox</h1>
      <address><time datetime="2019-11-07T00:00:00+00:00">07 Nov 2019</time> by <a rel="author">Molly Campbell</a></address>
      <figure>
        <img src="https://cdn.technologynetworks.com/tn/images/thumbs/jpeg/640_360/a-highly-precise-cas9-enzyme-sacas9-hf-is-added-to-the-crispr-toolbox-326965.jpg?v=10440709"/>
      </figure>
      <p>
        <i>The newly developed SaCas9-HF can be applied in gene editing which requires high precision. Credit: City University of Hong Kong.</i>
      </p>
      <p>Read Time:</p>
      <p><i>A team of scientists from City University of Hong Kong (CityU) and the Karolinska Institute has created a novel protein that can increase the target accuracy in genome editing. Their findings are published in the journal </i><a href="https://www.pnas.org/content/116/42/20969"><i>Proceedings of the National Academy of Sciences (PNAS)</i></a><i>.<br/></i><i><br/></i><b>Meet CRISPR<br/></b><b><br/></b>The gene editing technology Clustered Regularly Interspaced Short Palindromic Repeats (CRISPR)-Cas9 looks set to revolutionize modern medicine, agriculture, and synthetic biology.<br/><br/>The ability to edit the genome in vivo offers the potential to develop novel gene therapies for diseases that currently lack viable treatment options. Several clinical trials are underway exploring the utility of CRISPR technology in treating specific cancers, blood disorders and eye diseases.<br/><br/>CRISPR-Cas9 as a gene editing tool is superior over other techniques due to its ease of use. In traditional gene therapy, additional copies of the "normal" gene are introduced into cells. Using CRISPR technology, this isnâ€™t necessary; CRISPR-Cas9 enters the cell and "repairs" the problematic gene by removing it or correcting it to restore normal physiological function.<br/><br/>There are different components to the CRISPR mechanism. Cas9 is the enzyme that flags and locates the problematic DNA throughout the genome, acting in a "hunting" fashion. However, the precision of Cas9 cannot always be established, and occasionally modifications of DNA at unintended places can occur. If CRISPR is to be utilized to repair faulty genes in patients, potential off-target genome editing could have serious adverse effects.</p>
      <related>
        <a href="https://www.technologynetworks.com/genomics/news/technoloy-networks-explores-the-crispr-revolution-an-interview-with-professor-jennifer-doudna-co-325876"/>
      </related>
      <p>There are currently two versions of the Cas9 enzyme commonly adopted in CRISPR research: SpCas9 (Cas9 nuclease from the bacteria Streptococcus pyogenes) and SaCas9 (Cas9 nuclease from Staphylococcus aureus). Both of these enzymes are limited in that they possess a certain level of imprecision. <br/><br/>Thus, scientists have endeavored to develop variants of both enzymes, with the aim being to increase their precision and reduce off-target effects. The issue with SpCas9 is that the modified variants are often too large to "fit" in the delivery system adopted for inserting gene therapies into patients, known as adeno-associated viral (AAV) vectors.<br/><br/>SaCas9 is advantageous over SpCas9 in that it can be easily packaged into the AAV vectors for delivering gene-editing contents in vivo. However, at present, there is no SaCas9 variant that possesses high accuracy in genome-wide editing. Until now.<br/><br/><b>Now meet SaCas9-HF<br/></b><b><br/></b>In the new study published in <a href="https://www.pnas.org/content/116/42/20969">Proceedings of the National Academy of Sciences (PNAS)</a>, a research team led by Zheng Zongli, Assistant Professor of Department of Biomedical Sciences at CityU and the Ming Wai Lau Centre for Reparative Medicine of the Karolinska Institute in Hong Kong, and Shi Jiahai, Assistant Professor of Department of Biomedical Sciences at CityU, has successfully engineered SaCas9-HF, a CRISPR Cas9 variant which has demonstrated high accuracy in genome-wide targeting in human cells without compromising on-target efficiency.<br/><br/>In the study, the scientists conducted an extensive evaluation of 24 targeted human genetic locations comparing the original (known as wild-type) SaCas9, and the new variant, SaCas9-HF. They discovered that for targets with highly similar sequences in the genome (and therefore often disposed to off-target editing by wild-type Cas9), SaCas9-HF decreased the off-target activity by ~90%. When assessing targets that had relatively less off-targeting editing by wild-type SaCas9, the SaCas9-HF enzyme produced little to no detectable off-target effects.<br/><br/>"Our development of this new SaCas9 provides an alternative to the wild-type Cas9 toolbox, where highly precise genome editing is needed. It will be particularly useful for future gene therapy using AAV vectors to deliver genome editing 'drug' in vivo and would be compatible with the latest 'prime editing' CRISPR platform, which can 'search-and-replace' the targeted genes," said Dr Zheng.<br/><br/><b>Reference:</b> Tan et al. 2019. Rationally engineered Staphylococcus aureus Cas9 nucleases with high genome-wide specificity. <i>Proceedings of the National Aacademy of Sciences (PNAS). </i>DOI: <a href="https://doi.org/10.1073/pnas.1906843116">https://doi.org/10.1073/pnas.1906843116</a></p>
    </article>
  </body>
</html>