<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8"/>
    <meta property="og:url" content="https://www.sciencedaily.com/releases/2019/11/191119143244.htm"/>
    <meta property="og:site_name" content="ScienceDaily"/>
    <meta property="article:published_time" content="2019-11-19T00:00:00+00:00"/>
    <meta property="og:title" content="Technique identifies T cells primed for certain allergies or infections"/>
    <meta property="og:description" content="Researchers can now identify T cells reactive to a particular target from a patient's cells, and to perform high-throughput single-cell RNA sequencing of those cells."/>
  </head>
  <body>
    <article>
      <h1>Technique identifies T cells primed for certain allergies or infections</h1>
      <h2>Researchers develop a method to isolate and sequence the RNA of T cells that react to a specific target</h2>
      <address>
        <time datetime="2019-11-19T00:00:00+00:00">19 Nov 2019</time>
      </address>
      <table bordered="">
        <tr>
          <td>Source:</td>
          <td>Massachusetts Institute of Technology</td>
        </tr>
        <tr>
          <td>Summary:</td>
          <td>Researchers can now identify T cells reactive to a particular target from a patient's cells, and to perform high-throughput single-cell RNA sequencing of those cells.</td>
        </tr>
      </table>
      <p>
        <b>When your immune system is exposed to a vaccine, an allergen, or an infectious microbe, subsets of T cells that can recognize a foreign intruder leap into action. Some of these T cells are primed to kill infected cells, while others serve as memory cells that circulate throughout the body, keeping watch in case the invader reappears.</b>
      </p>
      <p>MIT researchers have now devised a way to identify T cells that share a particular target, as part of a process called high-throughput single-cell RNA sequencing. This kind of profiling can reveal the unique functions of those T cells by determining which genes they turn on at a given time. In a new study, the researchers used this technique to identify T cells that produce the inflammation seen in patients with peanut allergies.</p>
      <p>In work that is now underway, the researchers are using this method to study how patients' T cells respond to oral immunotherapy for peanut allergies, which could help them determine whether the therapy will work for a particular patient. Such studies could also help guide researchers in developing and testing new treatments.</p>
      <p>"Food allergies affect about 5 percent of the population, and there's not really a clear clinical intervention other than avoidance, which can cause a lot of stress for families and for the patients themselves," says J. Christopher Love, the Raymond A. and Helen E. St. Laurent Professor of Chemical Engineering and a member of MIT's Koch Institute for Integrative Cancer Research. "Understanding the underlying biology of what drives these reactions is still a really critical question."</p>
      <p>Love and Alex K. Shalek, who is the Pfizer-Laubach Career Development Associate Professor at MIT, an associate professor of chemistry, a core member of MIT's Institute for Medical Engineering and Science (IMES), and an extramural member of the Koch Institute, are the senior authors of the study, which appears today in <i>Nature Immunology</i>. The lead authors of the paper are graduate student Ang Andy Tu and former postdoc Todd Gierahn.</p>
      <h4>Extracting information</h4>
      <p>The researchers' new method builds on their previous work developing techniques for rapidly performing single-cell RNA sequencing on large populations of cells. By sequencing messenger RNA, scientists can discover which genes are being expressed at a given time, giving them insight into individual cells' functions.</p>
      <p>Performing RNA sequencing on immune cells, such as T cells, is of great interest because T cells have so many different roles in the immune response. However, previous sequencing studies could not identify populations of T cells that respond to a particular target, or antigen, which is determined by the sequence of the T cell receptor (TCR). That's because single-cell RNA sequencing usually tags and sequences only one end of each RNA molecule, and most of the variation in T cell receptor genes is found at the opposite end of the molecule, which doesn't get sequenced.</p>
      <p>"For a long time, people have been describing T cells and their transcriptome with this method, but without information about what kind of T cell receptor the cells actually have," Tu says. "When this project started, we were thinking about how we could try to recover that information from these libraries in a way that doesn't obscure the single-cell resolution of these datasets, and doesn't require us to dramatically change our sequencing workflow and platform."</p>
      <p>In a single T cell, RNA that encodes T cell receptors makes up less than 1 percent of the cell's total RNA, so the MIT team came up with a way to amplify those specific RNA molecules and then pull them out of the total sample so that they could be fully sequenced. Each RNA molecule is tagged with a barcode to reveal which cell it came from, so the researchers could match up the T cells' targets with their patterns of RNA expression. This allows them to determine which genes are active in populations of T cells that target specific antigens.</p>
      <p>"To put the function of T cells into context, you have to understand what it is they're trying to recognize," Shalek says. "This method lets you take existing single-cell RNA sequencing libraries and pull out relevant sequences you might want to characterize. At its core, the approach is a straightforward strategy for extracting some of the information that's hidden inside of genome-wide expression profiling data."</p>
      <p>Another advantage of this technique is that it doesn't require expensive chemicals, relies on equipment that many labs already have, and can be applied to many previously processed samples, the researchers say.</p>
      <h4>Analyzing allergies</h4>
      <p>In the <i>Nature Immunology</i> paper, the researchers demonstrated that they could use this technique to pick out mouse T cells that were active against human papilloma virus, after the mice had been vaccinated against the virus. They found that even though all of these T cells reacted to the virus, the cells had different TCRs and appeared to be in different stages of development — some were very activated for killing infected cells, while others were focused on growing and dividing.</p>
      <p>The researchers then analyzed T cells taken from four patients with peanut allergies. After exposing the cells to peanut allergens, they were able to identify T cells that were active against those allergens. They also showed which subsets of T cells were the most active, and found some that were producing the inflammatory cytokines that are usually associated with allergic reactions.</p>
      <p>"We can now start to stratify the data to reveal what are the most important cells, which we were not able to identify before with RNA sequencing alone," Tu says.</p>
      <p>Love's lab is now working with researchers at Massachusetts General Hospital to use this technique to track the immune responses of people undergoing oral immunotherapy for peanut allergies — a technique that involves consuming small amounts of the allergen, allowing the immune system to build up tolerance to it.</p>
      <p>In clinical trials, this technique has been shown to work in some but not all patients. The MIT/MGH team hopes that their study will help identify factors that could be used to predict which patients will respond best to the treatment.</p>
      <p>"One would certainly like to have a better sense of whether an intervention is going to be successful or not, as early as possible," Love says.</p>
      <p>This strategy could also be used to help develop and monitor immunotherapy treatments for cancer, such as CAR-T cell therapy, which involves programming a patient's own T cells to target a tumor. Shalek's lab is also actively applying this technique with collaborators at the Ragon Institute of MGH, MIT and Harvard to identify T cells that are involved in fighting infections such as HIV and tuberculosis.</p>
      <p>The research was funded by the Koch Institute Support (core) Grant from the National Institutes of Health, the Koch Institute Dana-Farber/Harvard Cancer Center Bridge Project, the Food Allergy Science Initiative at the Broad Institute of MIT and Harvard, the Arnold and Mabel Beckman Foundation, a Searle Scholar Award, a Sloan Research Fellowship in Chemistry, the Pew-Stewart Scholars program, and the National Institutes of Health.</p>
      <p>make a difference: sponsored opportunity</p>
      <details open="">
        <summary>
          <b>Story Source:</b>
        </summary>
        <p><a href="http://news.mit.edu/2019/t-cells-target-allergies-infections-1119">Materials</a> provided by <a href="http://web.mit.edu/"><b>Massachusetts Institute of Technology</b></a>. Original written by Anne Trafton. <br/><i>Note: Content may be edited for style and length.</i></p>
      </details>
      <details open="">
        <summary><b>Journal Reference</b>:</summary>
        <ol>
          <li>Ang A. Tu, Todd M. Gierahn, Brinda Monian, Duncan M. Morgan, Naveen K. Mehta, Bert Ruiter, Wayne G. Shreffler, Alex K. Shalek, J. Christopher Love. <b>TCR sequencing paired with massively parallel 3′ RNA-seq reveals clonotypic T cell signatures</b>. <i>Nature Immunology</i>, 2019; 20 (12): 1692 DOI: <a href="http://dx.doi.org/10.1038/s41590-019-0544-5">10.1038/s41590-019-0544-5</a></li>
        </ol>
      </details>
      <details>
        <summary><b>Cite This Page</b>:</summary>
        <table bordered="">
          <tr>
            <td>
              <a href="https://www.sciencedaily.com/releases/2019/11/191119143244.htm#citation_mla">MLA</a>
            </td>
            <td>Massachusetts Institute of Technology. "Technique identifies T cells primed for certain allergies or infections: Researchers develop a method to isolate and sequence the RNA of T cells that react to a specific target." ScienceDaily. ScienceDaily, 19 November 2019. &lt;www.sciencedaily.com/releases/2019/11/191119143244.htm&gt;.</td>
          </tr>
          <tr>
            <td>
              <a href="https://www.sciencedaily.com/releases/2019/11/191119143244.htm#citation_apa">APA</a>
            </td>
            <td>Massachusetts Institute of Technology. (2019, November 19). Technique identifies T cells primed for certain allergies or infections: Researchers develop a method to isolate and sequence the RNA of T cells that react to a specific target. <i>ScienceDaily</i>. Retrieved November 20, 2019 from www.sciencedaily.com/releases/2019/11/191119143244.htm</td>
          </tr>
          <tr>
            <td>
              <a href="https://www.sciencedaily.com/releases/2019/11/191119143244.htm#citation_chicago">Chicago</a>
            </td>
            <td>Massachusetts Institute of Technology. "Technique identifies T cells primed for certain allergies or infections: Researchers develop a method to isolate and sequence the RNA of T cells that react to a specific target." ScienceDaily. www.sciencedaily.com/releases/2019/11/191119143244.htm (accessed November 20, 2019).</td>
          </tr>
        </table>
      </details>
      <related>
        <h4>Related Stories</h4>
        <a href="https://www.sciencedaily.com/releases/2019/04/190422151026.htm"/>
        <a href="https://www.sciencedaily.com/releases/2017/07/170728092539.htm"/>
        <a href="https://www.sciencedaily.com/releases/2017/05/170511165359.htm"/>
        <a href="https://www.sciencedaily.com/releases/2015/12/151221111440.htm"/>
      </related>
      <related>
        <h4>Related Terms</h4>
        <a href="https://www.sciencedaily.com/terms/gene_therapy.htm"/>
        <a href="https://www.sciencedaily.com/terms/adult_stem_cell.htm"/>
        <a href="https://www.sciencedaily.com/terms/natural_killer_cell.htm"/>
        <a href="https://www.sciencedaily.com/terms/stem_cell.htm"/>
        <a href="https://www.sciencedaily.com/terms/somatic_cell.htm"/>
        <a href="https://www.sciencedaily.com/terms/cell_(biology).htm"/>
        <a href="https://www.sciencedaily.com/terms/monoclonal_antibody_therapy.htm"/>
        <a href="https://www.sciencedaily.com/terms/t_cell.htm"/>
      </related>
    </article>
  </body>
</html>