<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8"/>
    <meta property="og:url" content="https://www.futurity.org/diarrhea-childrens-health-e-coli-2219902-2/"/>
    <meta property="og:site_name" content="Futurity"/>
    <meta property="article:published_time" content="2019-11-25T14:57:05+00:00"/>
    <meta property="og:title" content="Lab tests may misdiagnose 50% of diarrhea’s bacterial causes in kids"/>
    <meta property="og:description" content="Conventional culture-based lab tests may misdiagnose as many as half of the microbial causes of diarrhea diseases in children"/>
  </head>
  <body>
    <article>
      <h1>Lab tests may misdiagnose 50% of diarrhea’s bacterial causes in kids</h1>
      <address><time datetime="2019-11-25T14:57:05+00:00">25 Nov 2019, 14:57</time> by <a rel="author">John Toon-Georgia Tech</a></address>
      <p>Conventional culture-based lab tests may misdiagnose as many as half of the microbial causes of diarrhea in children, a new study shows.</p>
      <p>Further, the study, which used samples from Ecuadorian children, also found that a common strain of the <i>E. coli</i> bacterium may be more virulent than previously believed.</p>
      <p>The research, which used multiple lines of evidence to determine the microbe causing changes in the gut microbiome during a <a href="https://www.futurity.org/diarrheal-disease-children-public-health-surveys-2027152-2/">diarrheal episode</a>, may be significant for applying new diagnostic technologies that could enable personalized medical treatments of intestinal diseases.</p>
      <p>The project is part of a study that integrates epidemiological, molecular, and metagenomic data to understand how enteric (food and waterborne) pathogens and the gut microbiome vary across an urban-rural gradient in Ecuador. Frequent illness can affect the growth and development of children during their critical early years.</p>
      <p>The new findings appear in <i><a href="https://doi.org/10.1128/AEM.01820-19">Applied and Environmental Microbiology</a></i>.</p>
      <h3>When is a microbe not a pathogen?</h3>
      <p>“We wanted to understand where the illnesses are coming from and what the consequences are for the development of the children,” says Kostas Konstantinidis, a professor in the School of Civil and Environmental Engineering at Georgia Tech.</p>
      <p>“Knowing more about the causative agent may allow us to prevent infections. This would be relevant not just for the developing world, but for children everywhere.”</p>
      <p>Findings about the likely causative agent of <a href="https://www.futurity.org/rotavirus-gut-microbiota-2182322-2/">enteric diseases</a> raise questions about long-held definitions of pathogenic agents, says Karen Levy, an associate professor at the Emory Rollins School of Public Health.</p>
      <p>Many so-called “pathogenic <i>E. coli</i>” infections are asymptomatic, not causing diarrhea, she notes. If health experts detect a microbe considered a pathogen in a stool sample, but the microbe doesn’t cause disease, is it a pathogen?</p>
      <p>“The approach we used in this analysis helps to not only detect whether or not the pathogenic <i>E. coli</i> is present in the stool, but also if it is the likely cause of diarrhea experienced by study subjects,” Levy says.</p>
      <p>“This holds promise that, in the future, we could use metagenomic approaches to diagnose not just the presence of the so-called ‘pathogenic <i>E. coli</i>,’ but also of the actual pathogenicity of these organisms.”</p>
      <h3>E. coli and diarrhea</h3>
      <p>Enteric infections often go undiagnosed in children, though they can in some circumstances be fatal. For the study, researchers collected more than 1,000 samples from infected children in Ecuador and analyzed them using traditional culture-based methods in which scientists place samples onto selective cell culture media, allow to grow, and then study them to determine the pathogen present.</p>
      <p>Researchers sent a subset of 30 samples to the Konstantinidis lab at Georgia Tech, where his team has developed new culture-independent genomics testing that can identify microbes that standard culture techniques may not detect. The researchers examined the total gut microbiome and its shifts during diarrheal infections to assess the effects of three specific pathogenic genotypes of <i>E. coli</i> on the indigenous gut microbiota.</p>
      <p>“We looked at the microbes in stool samples using culture-independent genomic techniques,” says Konstantinidis, who also has a faculty position in the School of Biological Sciences.</p>
      <p>“We took the DNA out of the samples, sequenced it, and used bioinformatics tools to see what microorganisms were there. By looking at the entire microbiome, we can get more precise information about the pathogens that are causing disease and their effects on the commensal microbes of the gastrointestinal tract: the microbiome. We found that the effects were different for different <i>E. coli</i> pathogens, which is important for diagnosis and distinguishing among them.”</p>
      <p>One major finding was that the metagenomics technique disagreed with the culture-based study. “In 50% of the cases where the lab suggested <i>E. coli</i> was the agent, we didn’t see evidence from metagenomics that this was the case,” Konstantinidis says. “We saw evidence that it was something else, potentially an enteric virus.”</p>
      <h3>Effect on growth and development</h3>
      <p>The researchers combined different diagnostic approaches in a way that offered a more complete picture of the microbiome, creating signatures that could potentially identify the microbial agent of the disease.</p>
      <p>“Compared to the traditional approach, our technique uses multiple lines of evidence, including relative pathogen abundance, population clonality level and detection of virulence factors, and effects on the gut’s natural microbiome that haven’t been used together before,” Konstantinidis says.</p>
      <p>The researchers also discovered that large amounts of co-eluted human DNA accompanied a strain known as diffuse adherent <i>E. coli</i> (DAEC) in the stool sample. Though scientists don’t generally consider DAEC a particularly virulent strain, the presence of human DNA suggests that the DAEC may have damaged epithelium cells in the gastrointestinal tract.</p>
      <p>“Understanding changes in the gut microbiome can help us understand how different strains cause distinct pathogenicity and symptoms such as the elution of high amount of human DNA and changes in the abundance of commensal microbiota,” Konstantinidis says. “If infections like this happen often, that could have implications for a child’s growth and development.”</p>
      <h3>Innovation in diarrhea diagnosis and treatment</h3>
      <p>The metagenomics technique could help provide a foundation for personalized medicine that would select therapies based on the specifics of the microbe—including its virulence and antibiotic resistance profiles. The study shows that the technology works and can be done quickly, but trained personnel are needed to make it more widely available.</p>
      <p>“In the next five or 10 years, we are going to see some big changes in which these new methodologies are adopted in the clinic, and that could lead, in the long term, to innovation in diagnosis and treatment,” Konstantinidis says.</p>
      <p>Future work will examine how environmental factors—such as water quality, presence of animals in the home, and the chemical environment—affect the microbiomes of small children. In this cohort of children from different geographic areas, the team will examine additional enteric pathogens, and follow children for their first two years of life. This will offer the opportunity to understand whether infections are cleared from the body—or if they continue to lurk in low levels.</p>
      <p>Additional coauthors are from Universidad San Francisco de Quito, Georgia Tech, and Emory. The National Institute of Allergy and Infectious Diseases funded the work. The contents are solely the responsibility of the author and do not necessarily represent the official views of the NIH.</p>
      <p>
        <i>Source: <a href="https://rh.gatech.edu/news/629100/metagenomics-unlocks-unknowns-diarrheal-disease-cases-children">Georgia Tech</a></i>
      </p>
      <footer><a href="https://doi.org/10.1128/AEM.01820-19">Original Study</a> DOI: 10.1128/AEM.01820-19</footer>
      <related>
        <h4>Related</h4>
        <a href="https://www.futurity.org/antibiotics-yanomami-microbiomes-904682/"/>
        <a href="https://www.futurity.org/bacteria-toxins-neighbors-993492-2/"/>
        <a href="https://www.futurity.org/rotavirus-gut-microbiota-2182322-2/"/>
      </related>
    </article>
  </body>
</html>