<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8"/>
    <meta property="og:url" content="https://www.technologynetworks.com/genomics/news/scooped-up-non-coding-dna-may-help-drive-glioblastoma-327548"/>
    <meta property="og:site_name" content="Technology Networks"/>
    <meta property="article:published_time" content="2019-11-22T00:00:00+00:00"/>
    <meta property="og:title" content="Scooped up Non-coding DNA May Help Drive Glioblastoma"/>
    <meta property="og:description" content="One of the ways a cancer-causing gene works up enough power to turn a normal cell into a cancer cell is by copying itself over and over, like a Xerox machine. Scientists have long noticed that when cancer-causing genes do that, they also scoop up some extra DNA into their copies."/>
  </head>
  <body>
    <article>
      <h1>Scooped up Non-coding DNA May Help Drive Glioblastoma</h1>
      <address>
        <time datetime="2019-11-22T00:00:00+00:00">22 Nov 2019</time>
      </address>
      <figure>
        <img src="https://cdn.technologynetworks.com/tn/images/thumbs/jpeg/640_360/scooped-up-non-coding-dna-may-help-drive-glioblastoma-327548.jpg?v=10462235"/>
      </figure>
      <p>
        <i>Credit: Pixabay</i>
      </p>
      <p>Read Time:</p>
      <p>One of the ways a cancer-causing gene works up enough power to turn a normal cell into a cancer cell is by copying itself over and over, like a Xerox machine. Scientists have long noticed that when cancer-causing genes do that, they also scoop up some extra DNA into their copies. But it has remained unclear whether the additional DNA helps drive cancer or is just along for the ride.<br/><br/>Using human glioblastoma brain tumor samples, researchers at University of California San Diego School of Medicine and Case Western Reserve University School of Medicine have now determined that all of that extra DNA is critical for maintaining a cancer-causing gene’s activation, and ultimately supporting a cancer cell’s ability to survive. Comparing those findings to a public database of patient tumor genetics, they also discovered that even if two different tumor types are driven by the same cancer-causing gene, the extra DNA may differ.<br/><br/>The study, published November 21, 2019 in <a href="https://www.cell.com/cell/fulltext/S0092-8674(19)31216-4"><i>Cell</i></a>, could explain why drugs will often work for some cancer types but not others.<br/><br/>“We’ve been targeting the cancer-causing gene for therapy, but it turns out we should also think about targeting the switches that are carried along with it,” said co-senior author Peter Scacheri, PhD, Gertrude Donnelly Hess Professor of Oncology at Case Western Reserve University School of Medicine and member of the Case Comprehensive Cancer Center.<br/><br/>When the human genome was first fully sequenced, many people were surprised to find it contained far fewer genes — segments of DNA that encode proteins — than expected. It turns out that the remainder of human DNA in the genome, the non-coding regions, play important roles in regulating and enhancing the protein-coding genes — turning them “on” and “off,” for example.<br/><br/>In this study, the researchers focused on one example cancer-causing gene, <i>EGFR</i>, which is particularly active in glioblastoma, an aggressive form of brain cancer, and other cancers. When copies of <i>EGFR</i> pile up in tumors, they tend to be in the form of circular DNA, separate from the chromosome.<br/><br/>“In 2004, I was the lead on the first clinical trial to test a small molecule inhibitor of <i>EGFR</i> in glioblastoma,” said co-senior author Jeremy Rich, MD, professor of medicine at UC San Diego School of Medicine and director of neuro-oncology and director of the Brain Tumor Institute at UC San Diego Health. “But it didn’t work. And here we are 15 years later, still trying to understand why brain tumors don’t respond to inhibitors of what seems to be one of the most important genes to make this cancer grow.”<br/><br/>The team took a closer look at the extra DNA surrounding <i>EGFR</i> circles in 9 of 44 different glioblastoma tumor samples donated by patients undergoing surgery. They discovered that the circles contained as many as 20 to 50 enhancers and other regulatory elements. Some of the regulatory elements had been adjacent to <i>EGFR</i> in the genome, but others were pulled in from other regions of the genome.<br/><br/>To determine the role each regulatory element plays, the researchers silenced them one at a time. They concluded that nearly every single regulatory element contributed to tumor growth.<br/><br/>“It looks like the cancer-causing gene grabs as many switches it can get its hands on ... co-opting their normal activity to maximize its own expression,” Scacheri said.<br/><br/>First author Andrew Morton, a graduate student in Scacheri’s lab, then searched a public database of patient tumor genetic information — more than 4,500 records covering nine different cancer types. He found that the team’s observation was not limited to <i>EGFR</i> and glioblastoma. Enhancers were amplified alongside cancer-causing genes in many tumors, most notably the <i>MYC</i> gene in medulloblastoma and <i>MYCN</i> in neuroblastoma and Wilms tumors.<br/><br/>“People thought that the high copy number alone explained the high activity levels of cancer-causing genes, but that’s because people weren’t really looking at the enhancers,” Morton said. “The field has been really gene-centric up to this point, and now we’re taking a broader view.”<br/><br/>Next, the researchers want to know if the diversity in regulatory elements across cancer types could also be helping tumors evolve and resist chemotherapy. They also hope to find a class of therapeutic drugs that inhibit these regulatory elements, providing another way to put the brakes on cancer-causing genes.<br/><br/>“This isn’t just a laboratory phenomenon, it’s information I need to better treat my patients,” said Rich, who is also a faculty member in the Sanford Consortium for Regenerative Medicine and Sanford Stem Cell Clinical Center at UC San Diego Health.<br/><br/>Reference: Morton et al. 2019. Functional Enhancers Shape Extrachromosomal Oncogene Amplifications. <i>Cell.</i> DOI: https://doi.org/10.1016/j.cell.2019.10.039. <br/><br/>This article has been republished from the following <a href="https://ucsdnews.ucsd.edu/pressrelease/non-coding-dna-located-outside-chromosomes-may-help-drive-glioblastoma">materials</a>. Note: material may have been edited for length and content. For further information, please contact the cited source.</p>
    </article>
  </body>
</html>